packages/neutrino-preset-karma/index.js

```diff
@@ -23,7 +23,12 @@ module.exports = (neutrino, opts = {}) => {
       }
     },
     frameworks: ['mocha'],
-    files: [tests],
+    files: [{
+      pattern: tests,
+      watched: false,
```

Comment:
It should never be watched... The compiled test files are watched and that is all that is needed. No need to additionally watch the test src files.